env-variables.html

```diff
@@ -194,7 +194,7 @@
 <li class="toctree-l4"><a class="reference internal" href="api/operations/collective-operations/alltoallv.html">Alltoallv</a></li>
 <li class="toctree-l4"><a class="reference internal" href="api/operations/collective-operations/barrier.html">Barrier</a></li>
 <li class="toctree-l4"><a class="reference internal" href="api/operations/collective-operations/broadcast.html">Broadcast</a></li>
-<li class="toctree-l4"><a class="reference internal" href="api/operations/collective-operations/group-call.html">Group Calls</a></li>
+<li class="toctree-l4"><a class="reference internal" href="api/operations/collective-operations/group-call.html">Group calls</a></li>
```

Comment:
Maybe I got something really wrong, but where exactly should it end up ?

I can see the target doc still has the capitalized version:


![image](https://github.com/user-attachments/assets/8df44392-7744-4393-8a3d-4b4a26e57b0e)

Reply:
I'm not sure whether this is for internal reference only. On the TOC, this ends up being capitalized just fine. 

https://www.intel.com/content/www/us/en/docs/oneccl/developer-guide-reference/2021-14/group-calls.html

Reply:
ok, thanks for comment